app/src/main/java/com/hadiyarajesh/notex/ui/folders/NoteFolderScreen.kt

```diff
@@ -0,0 +1,257 @@
+package com.hadiyarajesh.notex.ui.folders
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.grid.GridCells
+import androidx.compose.foundation.lazy.grid.LazyVerticalGrid
+import androidx.compose.foundation.lazy.grid.items
+import androidx.compose.foundation.lazy.grid.rememberLazyGridState
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.navigation.NavController
+import androidx.paging.compose.collectAsLazyPagingItems
+import com.hadiyarajesh.notex.R
+import com.hadiyarajesh.notex.database.entity.Folder
+import com.hadiyarajesh.notex.database.model.FolderType.NoteFolder
+import com.hadiyarajesh.notex.ui.navigation.Screens
+import java.time.Instant
+
+@Composable
+fun NoteFolderScreen(
+    modifier: Modifier = Modifier,
+    navController: NavController,
+    folderViewModel: NoteFolderViewModel
+) {
+    val scope = rememberCoroutineScope()
+    val context = LocalContext.current
+    val folders = remember { folderViewModel.getFolders(NoteFolder) }.collectAsLazyPagingItems()
+
+    val data = listOf(
```

Comment:
Fetch data from actual db, not static ones. 
If you need to test the UI, create your local branch an test but the code you push here should contains only relavant code.